packages/graphiql/src/components/QueryEditor.js

```diff
@@ -143,6 +143,12 @@ export class QueryEditor extends React.Component {
           }
         },
 
+        'Shift-Ctrl-B': () => {
```

Comment:
As per https://github.com/graphql/graphiql/issues/905#issuecomment-526861134 we're probably going to move to Monaco at some point; so if `Shift-Alt-F` works in Firefox can we use that instead please? Means less to re-learn in future :)

Also please can you add a comment above this explaining why this change was made?

```suggestion
        /* Shift-Ctrl-P is hard-coded in Firefox for private browsing mode, so add an alternative: */
        'Shift-Alt-F': () => {
```

Reply:
@benjie Yes understandable. I will make the requested changes